src/nvim/terminal.c

```diff
@@ -232,7 +232,7 @@ static int parse_osc8(VTermStringFragment frag, int *attr)
 {
   // Parse the URI from the OSC 8 sequence and add the URL to our URL set.
   // Skip the ID, we don't use it (for now)
-  size_t i = 0;
+  int i = 0;
```

Comment:
👀 is this temporary change or is it explained somewhere?

Reply:
I needed to change the type of `flag.len` to `int`, which causes "Comparison of integers of different signs" if I don't change this to `int`. I can just cast it to `int` every time instead if you prefer.

Reply:
So not temporary. I can keep it like this or revert to `size_t` and cast `flag.len` to `size_t`, both should work equally well. Lemme know if you have a preference.